cmd/client/main.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	post := flag.String("d", "", "data to POST")
+	help := flag.Bool("h", false, "usage help")
+	flag.Parse()
+
+	if *help || len(flag.Args()) != 2 {
+		fmt.Fprintln(os.Stderr, os.Args[0], "is a consumer application for the wifi-ap snap (see https://docs.ubuntu.com/core/en/stacks/network/wifi-ap/docs/index)")
+		fmt.Fprintln(os.Stderr, "	it communicates with wifi-ap via its control interface")
```

Comment:
.. is an example client application for the wifi-ap snap (...). It communicates with the wifi-ap snap over the provided unix domain socket. For example you can call its configuration endpoint  like

$ ...